app/models/zone.rb

```diff
@@ -229,6 +228,10 @@ def active?
     miq_servers.any?(&:active?)
   end
 
+  def none_miq_server_started?
```

Comment:
maybe `any_started_miq_servers?` and use .any?

The caller can do unless `any_started_miq_servers?`